.github/workflows/gh-pages.yml

```diff
@@ -82,3 +82,34 @@ jobs:
       - name: Push
         run: |-
           git push origin gh-pages:gh-pages
+
+  benchmark:
```

Comment:
Not sure this is needed in the `gh-pages` workflow?

Reply:
I had separated the workflows that runs every PR in the `main` workflow and the workflow that updates the charts in the gh-pages. I have consolidated them though so will remove.